vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj

```diff
@@ -75,6 +73,7 @@
       <AdditionalOptions>/await %(AdditionalOptions) /bigobj</AdditionalOptions>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard>stdcpp17</LanguageStandard>
+      <CallingConvention>Cdecl</CallingConvention>
     </ClCompile>
```

Comment:
why is this needed?

Reply:
It is not.
All VCPROJ files that import `React.Cpp.props` should not need to set `LanguageStandard` nor `CallingConvention`, etc.

Reply:
React.Cpp.props does not set CallingConvention.

Reply:
The only props file that explicilty sets CallingConvention is x86.props, which sets StdCall, which is not compatible with the unit test framework.

Reply:
I'll remove LanguageStandard as it's inhereited, but until #4096, we still need this here so x86 builds.

Reply:
> The only props file that explicilty sets CallingConvention is x86.props, which sets StdCall, which is not compatible with the unit test framework.

Yeah, `x86.props` ends up imported by `React.Cpp`.
Makes sense to set the calling convention in GTest-dependant projects.